x-pack/plugins/reporting/server/browsers/install.js

```diff
@@ -34,8 +34,8 @@ export async function installBrowser(logger, browserConfig, browserType, install
   const rawChecksum = await md5(binaryPath).catch(() => '');
 
   if (rawChecksum !== pkg.rawChecksum) {
-    logger.debug(`Extracting ${browserType} to ${binaryPath}`);
     const archive = path.join(browser.paths.archivesPath, pkg.archiveFilename);
+    logger.debug(`Extracting [${archive}] to [${binaryPath}]`);
```

Comment:
Hm, any thoughts that logging the absolute path might be a security concern?

Reply:
I checked with some devs and was told it's ok, and that we do it elsewhere in Kibana